manim/mobject/mobject.py

```diff
@@ -557,47 +629,117 @@ def update(self, dt=0, recursive=True):
                 submob.update(dt, recursive)
         return self
 
-    def get_time_based_updaters(self):
+    def get_time_based_updaters(self) -> "list[Callable]":
```

Comment:
add `from typing import List` and then
```suggestion
    def get_time_based_updaters(self) -> List[Callable]:
```